### PR TITLE
Summer sprint 2020

### PR DESCRIPTION
**Pull request by karenmajewicz**
Quick pass at a blog post. May have errors.

### CODE REVIEW

File: _posts/2020-09-15-summer-sprint-2020.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: GeoBlacklight 3.0 is Here!
+date: 2020-09-15 12:00:00
+categories: blog
+author: Karen Majewicz
+snippet: 'The GeoBlacklight Summer Community Sprint Activities## 2019 Winter Codesprint'
```
Comment (mejackreed) [inline _posts/2020-09-15-summer-sprint-2020.md:7]: Should this be updated? I see 2019 Winter

File: _posts/2020-09-15-summer-sprint-2020.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Our [Summer Community Sprint](https://github.com/geoblacklight/geoblacklight/projects/8) concluded in August 2020 and resulted in a [new version of GeoBlacklight](https://github.com/geoblacklight/geoblacklight/releases)! Version 3 brings us the long awaited multiple downloads option, support for Rails 6, and many improvements for accessibility. 
+
+### Notable Activities and Enhancements
+
+* MULTIPLE DOWNLOADS: To learn more about how to format metadata for multiple downloads, review the Wiki here: https://github.com/geoblacklight/geoblacklight/wiki/multiple-download-links
+
+* NEW NAME FOR THE DEFAULT BRANCH: We changed the name of the default branch to “Main.” There is a growing consensus in the development world for projects to change the default branch of their code in GitHub away from "master," as this is a loaded, potentially offensive, term. Read more about it here, including comments from our community: https://github.com/geoblacklight/geoblacklight/issues/940
+
+* ICONS: We deprecated the geoblacklight-icons project, which relied upon font-awesome for the icons. Now, new  SVG images can be submitted directly to the GeoBlacklight repository.
```
Comment (mejackreed) [inline _posts/2020-09-15-summer-sprint-2020.md:20]:
```suggestion
* ICONS: We deprecated the geoblacklight-icons project, which relied upon font-awesome for the icons. Now, new  SVG images can be submitted directly to the GeoBlacklight repository or added in your own GeoBlacklight application.
```

**Review by mejackreed — CHANGES_REQUESTED** [review]
Thanks for doing this, looks really good! Just a few minor questions/comments

File: _posts/2020-09-15-summer-sprint-2020.md (hunk 1 above)
Comment (karenmajewicz) [reply _posts/2020-09-15-summer-sprint-2020.md]: deleted in last commit

File: _posts/2020-09-15-summer-sprint-2020.md (hunk 2 above)
Comment (karenmajewicz) [reply _posts/2020-09-15-summer-sprint-2020.md]: added in last commit. Thanks!

**Review by andrewbattista — APPROVED** [review]
Look great to me ! Thanks @karenmajewicz

**Review by mejackreed — APPROVED** [review]

**Pull request merged**